views/default/admin/users/newest/time_cell.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Render a TD element for an item
+ *
+ * @uses $vars['item']      The item being rendered
+ * @uses $vars['item_vars'] Vars received from the page/components/table view
+ * @uses $vars['type']      The item type or ""
+ */
+
+if (empty($vars['item'])) {
+	// requested as an admin page
+	return;
+}
+
+$time = elgg_view('output/friendlytime', [
+	'time' => $vars['item']->time_created,
+]);
+
+?>
+<td style="white-space: nowrap"><?= $time ?></td>
```

Comment:
No sure it's a good approach for responsive tables. Something like `overflow-wrap: break-word;`
https://css-tricks.com/snippets/css/prevent-long-urls-from-breaking-out-of-container/